tools/license2rtf.js

```diff
@@ -1,15 +1,16 @@
+'use strict';
 
 var assert = require('assert'),
-    Stream = require('stream'),
-    inherits = require('util').inherits;
+  Stream = require('stream'),
+  inherits = require('util').inherits;
```

Comment:
Btw, eslint has a rule that could be used to forbid multiline declarations: [one-var](http://eslint.org/docs/rules/one-var).